lfs/lfs.go

```diff
@@ -103,9 +103,10 @@ func init() {
 
 	LocalWorkingDir, LocalGitDir, err = resolveGitDir()
 	if err == nil {
-		LocalMediaDir = filepath.Join(LocalGitDir, "lfs", "objects")
+		LocalGitStorageDir = resolveGitStorageDir(LocalGitDir)
+		LocalMediaDir = filepath.Join(LocalGitStorageDir, "lfs", "objects") // objects across all worktrees
 		LocalLogDir = filepath.Join(LocalMediaDir, "logs")
-		TempDir = filepath.Join(LocalGitDir, "lfs", "tmp")
+		TempDir = filepath.Join(LocalGitDir, "lfs", "tmp") // temp files per worktree
```

Comment:
Should these var comments go next to the declaration like the one you added for `LocalGitStorageDir`?